autoload/lsp/completion.vim

```diff
@@ -170,6 +172,25 @@ export def CompletionReply(lspserver: dict<any>, cItems: any)
     CompletionUltiSnips(prefix, items)
   endif
 
+  if opt.lspOptions.useBufferCompletion
+      # add completion from current buf
```

Comment:
Can you move the code inside this if statement into a separate function?

Reply:
add fix in https://github.com/yegappan/lsp/pull/284/commits/b48c94b90e9da99d6fa271f0eba8400cb5d11d48